test/service/test_schema_validation.py

```diff
@@ -0,0 +1,24 @@
+from furl import (
+    furl,
+)
+from jsonschema import (
+    Draft4Validator,
```

Comment:
Why did you switch from Draft 7 to Draft 4?

Reply:
Open API 3.0.3 is based on draft‑wright‑json‑schema‑00 , which has a metaschema identifier of `draft-04`
https://json-schema.org/specification-links.html
https://swagger.io/specification/#data-types

I had used the incorrect one the first time around.